libs/utils/src/lib.rs

```diff
@@ -67,7 +67,7 @@ macro_rules! failpoint_sleep_millis_async {
         let should_sleep: Option<std::time::Duration> = (|| {
             fail::fail_point!($name, |v: Option<_>| {
                 let millis = v.unwrap().parse::<u64>().unwrap();
-                Some(Duration::from_millis(millis))
+                Some(std::time::Duration::from_millis(millis))
```

Comment:
I think this is the right syntax?

```suggestion
                Some(::std::time::Duration::from_millis(millis))
```

Reply:
Probably a good one in a macro in case someone redefines `std`, like: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=95d864ac11e29f6c3ab61163d80e3ff9

Reply:
Ugh, I guess you're right. In principle, if we go down that route, we should fully-qualify _everything_ in the macro, including `tracing`, `tokio`, `Option`, `Some`, and `None`. You'd really want to move the code out of the macro into regular functions as much as possible at that point.

I went ahead and did that, see updated version. In hindsight this was probably not worth the trouble, we should've just added a short comment to warn about that danger and move one. If someone shadows `std`, they are either looking for trouble, or they have a good reason for it and they are on purpose redirecting calls to standard functions. But now that I wrote the code, might as well commit the "proper" fix. I don't think it's any more or less readable than the original.